mpsl/CHANGELOG.rst

```diff
@@ -9,6 +9,15 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
+Master branch
+*************
+
+Bug fixes
+=========
+
+* Fixed an issue where clock configuration option :c:member:`skip_wait_lfclk_started` in :c:struct:`mpsl_clock_lfclk_cfg_t`
+  might have not worked as expected with nRF5340 devices (DRGN-15223).
```

Comment:
```suggestion
* Fixed an issue where the clock configuration option :c:member:`mpsl_clock_lfclk_cfg_t.skip_wait_lfclk_started` in :c:struct:`mpsl_clock_lfclk_cfg_t` did not work as expected with nRF5340 devices (DRGN-15223).
```
Updated the wording, fixed the link to the struct member (need to test if this works), and removed the line break (one sentence per line).

Reply:
Great, but it displays the full "path" now and not only the member name - so no need to specify the struct in addition:
```suggestion
Fixed an issue where the clock configuration option :c:member:`mpsl_clock_lfclk_cfg_t.skip_wait_lfclk_started` did not work as expected with nRF5340 devices (DRGN-15223).
```

Reply:
Fixed